tests/unit/validation_test.cpp

```diff
@@ -336,3 +340,71 @@ TEST_F(WalletTxValidationTest, sign_verify_compact) {
     ASSERT_FALSE(
         cbdc::transaction::validation::check_attestations(ctx, m_pubkeys, 2));
 }
+
```

Comment:
I think this should be split into its own file.

Reply:
Do you mean the `MinterValidationTest`?  I actually had it as a separate file initially.  I can change it back to that.

Reply:
Yeah, I think it makes sense to be standalone.